packages/cloud_firestore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.8+1
```

Comment:
I think it would be 0.9.9 rather than 0.9.8+1 since a behavior change is evident